contacts_theme.libraries.yml

```diff
@@ -22,4 +22,6 @@ action.group:
     state:
       css/contacts/action-group.css: {}
   js:
+    js/util.js: {}
```

Comment:
util.js needs to be part of a separate library that we are unlikely to include directly...